mail_base/README.rst

```diff
@@ -0,0 +1,11 @@
+Mail Base
+=========
+
+Module contain common code for other mail modules
```

Comment:
Due to odoo restrictions, module makes initialization again. That is browser loads emoji and other chat data twice. This is the only way to make Mail feature extendable.